src/packages/v4/components/modals/V4TokenHoldersModal/DownloadTokenHoldersModal.tsx

```diff
@@ -119,6 +128,7 @@ export function DownloadTokenHoldersModal({
           <Trans>Block number</Trans>
         </label>
         <FormattedNumberInput
+          disabled // TODO disabled temporarily. Need to implement participants query by block number via bendystraw
```

Comment:
this is disabled for now. will re-enable once i add a thing (we'll need to make a direct sql query to bendystraw for this to work)